packages/amqp/lib/AbstractAmqpQueuePublisher.ts

```diff
@@ -0,0 +1,23 @@
+import type { Options } from 'amqplib/properties'
+
+import { AbstractAmqpPublisher } from './AbstractAmqpPublisher'
+
+export type AmqpQueueMessageOptions = {
+  publishOptions: Options.Publish
+}
+
+const NO_PARAMS: AmqpQueueMessageOptions = {
+  publishOptions: {},
+}
+
+export class AbstractAmqpQueuePublisher<
```

Comment:
This is a replacement for previously existing AbstractAmqpPublisher.

It's a breaking change.